Utils/ReleaseMaker.psm1

```diff
@@ -93,8 +93,9 @@ function New-ReleaseBuild
     try
     {
         if ( test-path out ) { remove-item out/ -recurse -force }
-        .\buildCoreClr.ps1 -Framework net451 -Configuration Release -Build
         .\buildCoreClr.ps1 -Framework net451 -Configuration PSV3Release -Build
+        .\buildCoreClr.ps1 -Framework net451 -Configuration PSV4Release -Build
```

Comment:
The essential bug was that it was not being built for version 4, the rest is just re-ordering that it builds v 3,4,5,6 in order for better readability

Reply:
the 1.16.1 version of Script Analyzer did not include a PSv4 directory. So we haven't shipped it for a long while. Has there been an issue with this? Is there any strong reason why we should start shipping a v4 version when we haven't previously?